sentences/nb/_common.yaml

```
@@ -350,6 +350,18 @@ lists:
     range:
       from: 1
       to: 100
+  timer_half:
+    values:
+      - in: "halv"
+        out: 30
+      - in: "halvt"
+        out: 30
+  timer_quarter:
+    values:
+      - in: "(1 |et |ett )kvarter"
```

The space should be outside the parentheses since it's always required
```suggestion
      - in: "(1|et|ett) kvarter"
```

---

*doh* of course